### PR TITLE
GTD-1: Added search option to template

### PR DESCRIPTION
**Pull request by lewisnyman**
See: alphagov/tech-docs-gem#28

### CODE REVIEW

File: template/Gemfile [hunk 1]
```diff
@@ -10,3 +10,6 @@ gem 'tzinfo-data', platforms: [:mswin, :mingw, :jruby]
 
 # Include the tech docs gem
 gem 'govuk_tech_docs'
+
+# Overrride middleman-search with our fork.
+gem 'middleman-search', :git => "git://github.com/alphagov/middleman-search.git"
```
[the remark this answers is not in the document]
Comment (lewisnyman) [reply template/Gemfile]: Thanks, good idea. I've added a link to the PR as a comment.

It seems that adding this line to the tech docs gem is not effective. It has to be added to the main project.

**Comment by lewisnyman** [comment]
@kevindew I've added a comment now. Thanks for the feedback.

**Comment by jonathanglassman** [comment]
@kevindew are you happy to approve?

**Comment by kevindew** [comment]
@lewisnyman no problem. 

Re

> It seems that adding this line to the tech docs gem is not effective. It has to be added to the main project.

I see, I'm guessing this is because we can't specify a fork in https://github.com/alphagov/tech-docs-gem/blob/master/govuk_tech_docs.gemspec#L32? This seems to create a strange situation where we need to treat this gem separately? Have we considered publishing the fork to resolve this?

I'll approve this now but it'd be good to get the gem situation resolved.

**Comment by lewisnyman** [comment]
> Have we considered publishing the fork to resolve this?
Not really, I don't feel like that's a decision I can make and I haven't raised it more widely yet. If we have to continue maintaining this fork for a substantial amount of time it feels like a logical next step.

**Pull request merged**